packages/amplify-velocity-template/package.json

```diff
@@ -23,7 +23,8 @@
     "Eward Song"
   ],
   "scripts": {
-    "test": "mocha tests --require should",
+    "test": "npm run test-sub",
+    "test-sub": "mocha tests --require should",
```

Comment:
Is there a reason for splitting this up?

Reply:
Yeah, all the other tests use jest and the -i in jest means "run in band" but -i in mocha means something else so I did this so the -i wouldn't be passed through to mocha